bin/systemml-standalone.py

```diff
@@ -21,180 +21,94 @@
 #-------------------------------------------------------------
 
 import os
-import shutil
-import sys
 from os.path import join, exists
+import argparse
+from utils import get_env, find_script_file, log4j_path, config_path
 
 
-# error help print
-def print_usage_and_exit():
-    this_script = sys.argv[0]
-    print('Usage: ' + this_script + ' <dml-filename> [arguments]')
-    sys.exit(1)
+def default_classpath(systemml_home):
+    """
+    Classpath information required for excution
 
+    return: String
+    Classpath location of build, library and hadoop directories
+    """
+    build_lib = join(systemml_home, 'target', '*')
+    lib_lib = join(systemml_home, 'target', 'lib', '*')
+    hadoop_lib = join(systemml_home, 'target', 'lib', 'hadoop', '*')
 
-# from http://stackoverflow.com/questions/1724693/find-a-file-in-python
-def find_file(name, path):
-    for root, dirs, files in os.walk(path):
-        if name in files:
-            return join(root, name)
-    return None
+    return build_lib, lib_lib, hadoop_lib
 
 
-if len(sys.argv) < 2:
-    print('Wrong usage')
-    print_usage_and_exit()
+def standalone_entry(nvargs, args, config, explain, debug, stats, gpu, f):
 
```

Comment:
Please add documentation for this function.
Also, do you think `standalone_execution_entry` or `standalone_mode_entry` is a better name?